include/zephyr/net/wifi_mgmt.h

```diff
@@ -112,6 +112,12 @@ enum net_request_wifi_cmd {
 	NET_REQUEST_WIFI_CMD_RTS_THRESHOLD_CONFIG,
 	/** WPS config */
 	NET_REQUEST_WIFI_CMD_WPS_CONFIG,
+	/** Start roaming */
+	NET_REQUEST_WIFI_CMD_START_ROAMING,
+	/** Neighbor report complete */
+	NET_REQUEST_WIFI_CMD_NEIGHBOR_REP_COMPLETE,
+	/** Specific scan */
+	NET_REQUEST_WIFI_CMD_SPEC_SCAN,
```

Comment:
can be confused with spectral scan (another feature in Wi-Fi). May be candidate scan?

Reply:
Updated to candidate_scan.